geometry_msgs/Point pos
---
bool finished